dts/bindings/usbc/usbc-port.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Common fields for USBC ports
+
+compatible: "usbc-port"
```

Comment:
I'm confused. It looks like this is meant to be an include file, not actually a compatible that is used within "real" devicetree files. Is that right? If so, this compatible should be removed.

Reply:
I updated the commit. This was not meant to be a common file